src/issues.ts

```diff
@@ -0,0 +1,35 @@
+import * as fs from "fs";
+
+const issuesFilePath = __dirname + "/res/osscameroon_issues.json";
+
+export interface Issue {
+    title?: string,
+    labels?: Array<String>,
+    author?: string,
+    issue?: string
+}
+
+const filterIssues = (items: Issue[]): Issue[] => {
+    items?.sort((a, b) => {
+        let y1 = a.labels?.filter((label) => label.toLowerCase().includes("yotas"))[0]?.split(" ")[0]
+        let y2 = b.labels?.filter((label) => label.toLowerCase().includes("yotas"))[0]?.split(" ")[0]
+
+        // if y1 or y2 is undefined or null give it a big number to put him in last
+        return Number(y1 ? y1 : '2000') - Number(y2 ? y2 : '2000');
```

Comment:
you can replace the syntax `y1 ? y1 : '2000'` by `y1 ?? '2000'` check out [this article](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) to learn more about.